packages/solid/src/reactive/observable.ts

```diff
@@ -1,17 +1,26 @@
-import { Accessor, createComputed, createSignal, onCleanup, Setter, untrack } from "./signal";
+import { Accessor, createComputed, createRoot, createSignal, getOwner, onCleanup, Setter, untrack } from "./signal";
+
+// Note: This will add Symbol.observable globally for all TypeScript users,
+// however, we are not polyfilling Symbol.observable. Ensuring the type for
+// this global symbol is present is necessary for `observable()` to be
+// properly typed for 3rd party library's like RXJS.
+declare global {
+  interface SymbolConstructor {
+    readonly observable: symbol;
```

Comment:
As the comment above this interface suggests, ensuring that a type for `Symbol.observable` exists is necessary for Solid's `observable()` function to be properly typed for RXJS. This is because RXJS's `from()` function expects an object with `{ [Symbol.observable](): Subscribabble<T> }`. Previously, `getSymbol()` didn't return `Symbol.observable` but instead returned `any` which breaks type interoperability.